packages/percy-snapshot-render-react/src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RedBox from 'redbox-react';
+
+export default function render(markup, el) {
```

Comment:
Created a new package that takes markup and a DOM element and uses React to render it. Since rendering is now happening client-side in Percy, if there's an error it may be hard to debug for users. So I've added `redbox-react` here, which will render a pretty formatted version of the stack trace if anything goes wrong.

![image](https://user-images.githubusercontent.com/733696/29498992-2f2b0fac-85d5-11e7-900d-83f9bd9ec6f4.png)